app/models/event.js

```diff
@@ -181,6 +181,21 @@ export default class Event extends ModelBase.extend(CustomPrimaryKeyMixin, {
 
   sessionsByState: computed('sessions', function() {
     return groupBy(this.sessions.toArray(), 'data.state');
+  }),
+
+  headererLocation: computed('locationName', 'online', function() {
+    if (this.locationName && this.online) {
+      return `In-Person Event and Online Event ${this.locationName}`;
+    }
```

Comment:
Codacy found an issue: [Closing curly brace does not appear on the same line as the subsequent block.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6259866)